mlserver/registry.py

```diff
@@ -196,7 +196,7 @@ async def get_model(self, version: str = None) -> MLModel:
 
     async def get_models(self) -> List[MLModel]:
         # NOTE: `.values()` returns a "view" instead of a list
-        return list(self._versions.values())
+        return list(v for k, v in self._versions.items() if k)
```

Comment:
Should we also add the previous `if self._default and not self._default.version` check to add the default model if it doesn't have a version?